csp-guru/datapackage.json

```diff
@@ -1,11 +1,161 @@
 {
-    "title": "A database of concentrated solar power plants of the world, for energy modellers and analysts.",
+    "title": "CSP.guru",
     "homepage": "https://csp.guru",
     "version": "2016-08-30",
+    "description": "A database of concentrated solar power plants of the world, for energy modellers and analysts.",
+    "long_description": "This datasets holds technical and economic data and  analysis on all operating concentrating solar power stations and of stations under construction (larger than 10 MW). For detailed description of the data collection procedure, please see the original article [Lilliestam et al. (2017)](https://doi.org/10.1038/nenergy.2017.94).\n## License and conditions\n\nThe licence of this data allows academic and all non-commercial use if you give appropriate credit, under the [Creative Commons Attribution-NonCommercial 4.0 International license](https://creativecommons.org/licenses/by-nc/4.0/).\n\nFor academic and professional use (presentations, journal articles, reports, etc):\nPlease cite the original source, the paper [Lilliestam et al. (2017)](https://doi.org/10.1038/nenergy.2017.94). Please also link to https://www.csp.guru if possible.\n## Contact\nFor questions or issues get in touch with [Johan Lilliestam](mailto:johan.lilliestam@usys.ethz.ch) or [Richard Thonig](mailto:richard.thonig@usys.ethz.ch).\n## Acknowledgements\nFunding for this work came from a European Research Council Consolidator Grant (grant number 313553).",
     "resources": [
         {
-            "path": "csp-guru.csv",
-            "hash": "0ed89d2fe7c96615ad70d79ce64693a8"
+            "path": "CSP.guru-2016-08-30.csv",
+            "hash": "0ed89d2fe7c96615ad70d79ce64693a8",
+            "schema": {
```

Comment:
Currently this throws warnings next to the error. I believe we should make this whole thing a Tabular Data Package. We would need to add `"profile": "tabular-data-package"` on the first level and `"profile": "tabular-data-resource"` on the level of the resource. 

There is an example here:
http://frictionlessdata.io/specs/tabular-data-package/#specification

Reply:
I just tried it and no, this does not remove the validation warning. I'd make this a tabular data package anyways.